daemon/src/engine/qcommon/qfiles.h

```diff
@@ -575,7 +575,7 @@ typedef struct
 	float  st[ 2 ];
 	float  lightmap[ 2 ];
 	vec3_t normal;
-	byte   color[ 4 ];
+	Color::Color32Bit color;
```

Comment:
Color32Bit is a standard layout type. Even though it isn't a trivial type, its default constructor does nothing but zero-initialize its members